docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -62,27 +62,25 @@ Start by checking out the proper Chipyard version. Run:
     # note: this may not be the latest release if the documentation version != "stable"
     git checkout |version|
 
-Next run the following script to create Chipyard's Conda environment including a pre-built RISC-V toolchain.
+Next run the following script to setup Chipyard completely with a specific toolchain.
 There are two toolchains, one for normal RISC-V programs called ``riscv-tools`` which is the one needed for most Chipyard use-cases, and another for Hwacha/Gemmini called ``esp-tools``.
 Run the following script based off which compiler you would like to use.
 
 .. code-block:: shell
 
     ./build-setup.sh riscv-tools # or esp-tools
 
-This script wraps around the conda environment initialization process and also runs the ``init-submodules-no-riscv-tools.sh`` and ``build-toolchain-extra.sh`` scripts.
+This script wraps around the conda environment initialization process, initializes all submodules (with the ``init-submodules-no-riscv-tools.sh`` script), installs a toolchain, and runs other setups.
+See ``./build-setup.sh --help`` for more details on what this does and how to disable parts of the setup.
 
 The ``init-submodules-no-riscv-tools.sh`` script will initialize and checkout all of the necessary git submodules.
 This will also validate that you are on a tagged branch, otherwise it will prompt for confirmation.
 When updating Chipyard to a new version, you will also want to rerun this script to update the submodules.
 Using ``git`` directly will try to initialize all submodules; this is not recommended unless you expressly desire this behavior.
```

Comment:
Remove references to `init-submodules`, preserve warning about not directly calling git.,